src/model/footsteps/hooks/index.ts

```diff
@@ -9,11 +9,15 @@ const loadFootstepsAudio = () => {
   return footsteps;
 };
 
+//オーディオインスタンス
+const footstepsAudio = loadFootstepsAudio();
```

Comment:
Since `new Audio` cannot be accessed on the server, some ingenuity is needed

fyi:
Initialize within the `playSounds` inside `useFootsteps`, which is certain to be executed on the client
Alternatively, if you want to persist the audio using `useRef`, initialize it inside `useFootsteps` but only when `isClient` is true

example:
```
  const footstepsAudio = useRef<HTMLAudioElement>();
  if (isClient && !footstepsAudio.current)
    footstepsAudio.current = loadFootstepsAudio();
```